tensorbay/client/version.py

```diff
@@ -891,3 +902,12 @@ def list_jobs(self, status: Optional[str] = None) -> PagingList[BasicSearchJob]:
             lambda offset, limit: self._generate_jobs(status, offset, limit),
             128,
         )
+
+    def delete_job(self, job_id: str) -> None:
```

Comment:
This methods in `BasicSearch` and `SquashAndMerge` seem identical, so why are they written twice? For the docstring?

Reply:
jobType is different

Reply:
But the job type can be accessed by the class constant.